templates/quickstart-jira-dc.template.yaml

```diff
@@ -589,6 +589,10 @@ Conditions:
     !Equals [!Ref InternetFacingLoadBalancer, 'true']
   GovCloudCondition:
     !Equals [!Ref 'AWS::Region', 'us-gov-west-1']
+  DBEngineAurora:
```

Comment:
What if instead of adding more conditions and `Ifs` for the database engine we will add `Mappings`? Something like this?

```
Mappings:
  DBEngine:
    PostgreSQL:
      "name": "rds_postgres"
    "Amazon Aurora PostgreSQL":
      "name": "aurora_postgres"
```

And then use `!FindInMap [ DBEngine, !Ref DBEngine, 'name' ]`

Reply:
I tried this initially and CFn template validation failed as the the key ("Amazon Aurora PostgreSQL") has spaces. Ideally the map is the better option, but given the amount of nested templates that need to change to make "Amazon Aurora PostgreSQL" not have any spaces, using conditionals turned out to be a more pragmatic choice

Reply:
TIL, thanks @theghostwhoforks